src/web/mjs/connectors/PixivComics.mjs

```diff
@@ -76,7 +76,7 @@ export default class PixivComics extends Connector {
 
     async _getPages(chapter) {
         const timestamp = new Date().toISOString().replace(/\.\d+Z$/, 'Z');
-        const hash = CryptoJS.SHA256(timestamp + '1P-7eSTKnyxRrsxXrvKcng4L68ju8RvuP38OhGAE-gE').toString(CryptoJS.enc.Hex);
+        const hash = CryptoJS.SHA256(timestamp + 'M7w5HORvvX-VP4tRj2CFQOQFPocBqLvHTIbhTU36UCo').toString(CryptoJS.enc.Hex);
```

Comment:
This one is still working, but already outdated
See https://comic.pixiv.net/viewer/stories/0
```
JSON.parse(document.querySelector('#__NEXT_DATA__').text).props.pageProps.salt;
```

Reply:
Nice catch.